diff --git a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
index 3db521c43..57508c1e1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
@@ -1110,10 +1110,6 @@ &hdmi_in_vopb {
 	status = "okay";
 };
 
-&hdmi_in_vopl {
-	status = "disable";
-};
-
 &dp_in_vopb {
 	status = "disable";
 };
