find . -name '*.dts*' | xargs sed -i -E 's/RK_FUNC_([0-9])/\1/'
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
index 1b2f605e2..3db521c43 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
@@ -983,7 +983,7 @@ hdmi {
 
 	i2s1 {
 		i2s_8ch_mclk: i2s-8ch-mclk {
-			rockchip,pins = <4 RK_PA0 RK_FUNC_1 &pcfg_pull_none>;
+			rockchip,pins = <4 RK_PA0 1 &pcfg_pull_none>;
 		};
 	};
 
@@ -1055,7 +1055,7 @@ leds_gpio: leds-gpio {
 
 	cam_pins {
 		cif_clkout_a: cif-clkout-a {
-			rockchip,pins = <2 11 RK_FUNC_3 &pcfg_pull_none>;
+			rockchip,pins = <2 11 3 &pcfg_pull_none>;
 		};
 
 		cif_clkout_a_sleep: cif-clkout-a-sleep {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dts b/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dts
index 927434051..b0ae3cc15 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dts
@@ -649,16 +649,16 @@ wifi_host_wake_l: wifi-host-wake-l {
 	i2s0 {
 		i2s0_8ch_bus: i2s0-8ch-bus {
 			rockchip,pins =
-				<3 24 RK_FUNC_1 &pcfg_pull_none>,
-				<3 25 RK_FUNC_1 &pcfg_pull_none>,
-				<3 26 RK_FUNC_1 &pcfg_pull_none>,
-				<3 27 RK_FUNC_1 &pcfg_pull_none>,
-				<3 30 RK_FUNC_1 &pcfg_pull_none>,
-				<3 31 RK_FUNC_1 &pcfg_pull_none>;
+				<3 24 1 &pcfg_pull_none>,
+				<3 25 1 &pcfg_pull_none>,
+				<3 26 1 &pcfg_pull_none>,
+				<3 27 1 &pcfg_pull_none>,
+				<3 30 1 &pcfg_pull_none>,
+				<3 31 1 &pcfg_pull_none>;
 		};
 
 		i2s_8ch_mclk: i2s-8ch-mclk {
-			rockchip,pins = <4 0 RK_FUNC_1 &pcfg_pull_none>;
+			rockchip,pins = <4 0 1 &pcfg_pull_none>;
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index f16bb6d69..e8dd9f18d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -2297,35 +2297,35 @@ sdio0_int: sdio0-int {
 		sdmmc {
 			sdmmc_bus1: sdmmc-bus1 {
 				rockchip,pins =
-					<4 RK_PB0 RK_FUNC_1 &pcfg_pull_up_8ma>;
+					<4 RK_PB0 1 &pcfg_pull_up_8ma>;
 			};
 
 			sdmmc_bus4: sdmmc-bus4 {
 				rockchip,pins =
-					<4 RK_PB0 RK_FUNC_1 &pcfg_pull_up_8ma>,
-					<4 RK_PB1 RK_FUNC_1 &pcfg_pull_up_8ma>,
-					<4 RK_PB2 RK_FUNC_1 &pcfg_pull_up_8ma>,
-					<4 RK_PB3 RK_FUNC_1 &pcfg_pull_up_8ma>;
+					<4 RK_PB0 1 &pcfg_pull_up_8ma>,
+					<4 RK_PB1 1 &pcfg_pull_up_8ma>,
+					<4 RK_PB2 1 &pcfg_pull_up_8ma>,
+					<4 RK_PB3 1 &pcfg_pull_up_8ma>;
 			};
 
 			sdmmc_clk: sdmmc-clk {
 				rockchip,pins =
-					<4 RK_PB4 RK_FUNC_1 &pcfg_pull_none_12ma>;
+					<4 RK_PB4 1 &pcfg_pull_none_12ma>;
 			};
 
 			sdmmc_cmd: sdmmc-cmd {
 				rockchip,pins =
-					<4 RK_PB5 RK_FUNC_1 &pcfg_pull_up_8ma>;
+					<4 RK_PB5 1 &pcfg_pull_up_8ma>;
 			};
 
 			sdmmc_cd: sdmmc-cd {
 				rockchip,pins =
-					<0 RK_PA7 RK_FUNC_1 &pcfg_pull_up>;
+					<0 RK_PA7 1 &pcfg_pull_up>;
 			};
 
 			sdmmc_wp: sdmmc-wp {
 				rockchip,pins =
-					<0 RK_PB0 RK_FUNC_1 &pcfg_pull_up_8ma>;
+					<0 RK_PB0 1 &pcfg_pull_up_8ma>;
 			};
 		};
 
